octoprint_mrbeam/filemanager/__init__.py

```diff
@@ -108,18 +118,24 @@ def _delete_files_by_age(self, files, num_files_to_keep=0):
     def _is_history_file(entry):
         _, extension = os.path.splitext(entry)
         extension = extension[1:].lower()
-        return extension == "mrb"
+        return extension in MrbFileManager.FILE_EXTENSIONS_HISTORY
 
     @staticmethod
     def _is_gcode_file(entry):
         _, extension = os.path.splitext(entry)
         extension = extension[1:].lower()
-        return extension == "gco"
+        return extension in MrbFileManager.FILE_EXTENSIONS_GCODE
 
     @staticmethod
     def _sanitize_content(file_name, content):
         _, extension = os.path.splitext(file_name)
-        if extension in (".svg", ".mrb", ".g", ".gco", ".gc", ".gcode", ".nc"):
+        extension = extension[1:].lower()
+        if (
+            extension
+            in MrbFileManager.FILE_EXTENSIONS_SVG
+            + MrbFileManager.FILE_EXTENSIONS_GCODE
+            + MrbFileManager.FILE_EXTENSIONS_HISTORY
```

Comment:
For the sake of this, it could be nice to have `MrbFileManager.FILE_EXTENSIONS_ALL`

Reply:
no, because we do want to exclude JPG and PNG